compatibility/transports/file/src/main/java/org/mule/compatibility/transport/file/ExpressionFilenameParser.java

```diff
@@ -23,21 +23,21 @@
  * <p/>
  * For example an xpath expression can be defined to pull a message id out of an xml message and use that as the file name -
  * <code>
- * #[xpath:/message/header/@id]
+ * #[mel:xpath:/message/header/@id]
```

Comment:
i know this is a comment, but it got me thinking if we need to keep xpath support in MEL after MEL goes to compatibility.